app/settings.py

```diff
@@ -83,7 +85,11 @@
     ]
 }
 
-CSRF_TRUSTED_ORIGINS = ["https://lantern.khoj.dev", "https://*.127.0.0.1"]
+CSRF_TRUSTED_ORIGINS = [
+    "https://lantern.khoj.dev",
```

Comment:
We should probably drive these configurations through environment variables.

Reply:
Yes or it should default to `https://app.khoj.dev`

Reply:
Let me pick up the renaming task in a separate PR.